runtime/src/bank.rs

```diff
@@ -979,6 +982,11 @@ pub struct Bank {
     /// The number of transactions processed without error
     transaction_count: AtomicU64,
 
+    /// The number of non-vote transactions processed without error since the creation of the
+    /// current ledger.  This value is not shared though the network, but is preserved between
+    /// validator restarts.
```

Comment:
I think the phrases "creation of the current ledger" and "between validator restarts" are both ambiguous. Here's a crack at alternative wording, but could be better.

```suggestion
    /// The number of non-vote transactions processed without error since the most recent boot
    /// from snapshot or genesis.  This value is not shared though the network nor retained within snapshots, but is preserved in Bank::new_from_parent.
```

Reply:
Thank you for double-checking.  Updated as you suggested.